packages/component-library-angular/src/document/component.ts

```diff
@@ -7,5 +7,6 @@ import { ChangeDetectionStrategy, Component } from '@angular/core';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class UtrechtDocument {
+  @Input() class?: string;
```

Comment:
I want to avoid adding global HTML attributes such `class` to our Angular components, so I would propose to remove this `@Input()`. If someone really needs `utrecht-document` with custom class names, then we should probably offer `<div utrecht-document>` instead.

Reply:
Thanks for this feedback, I will remove it
I found how to add a modifier class in Angular without using the `@Input() class`, actually use 
`<div utrecht-document [class]="custom-class">`